docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -41,6 +41,8 @@ Note: *hostPath* volume types created by kube play is given an SELinux shared la
 
 Note: To set userns of a pod, use the **io.podman.annotations.userns** annotation in the pod/deployment definition. This can be overridden with the `--userns` flag.
 
+Note: Use the **io.podman.annotations.volumes-from** annotation to bind mount volumes of one container to another. You can mount volumes from multiple source containers to a target container. The source containers that belong to the same pod must be defined before the source container in the kube YAML. The format is `io.podman.annotations.volumes-from/targetContainer: "sourceContainer1:mountOpts1;sourceContainer2:mountOpts2;..."`.
```

Comment:
We need to be careful with the example here because the length of the value of the annotation is limited to 253 characters.

Reply:
Any specific change do you want here?

Reply:
Updated.